frontend/src/properties/PropertiesWebSocketContainer.tsx

```diff
@@ -88,9 +88,13 @@ export const PropertiesWebSocketContainer = ({
    * Displays an other form if the selection indicates that we should display another properties view.
    */
   useEffect(() => {
-    if (currentSelection?.id !== selection?.id) {
-      const isRepresentation = registry.isRepresentation(selection.kind);
-      const switchSelectionEvent: SwitchSelectionEvent = { type: 'SWITCH_SELECTION', selection, isRepresentation };
+    if (selection.entries.length > 0 && selection.entries[0].id !== currentSelection?.id) {
+      const isRepresentation = registry.isRepresentation(selection.entries[0].kind);
+      const switchSelectionEvent: SwitchSelectionEvent = {
+        type: 'SWITCH_SELECTION',
+        selection: selection.entries[0],
+        isRepresentation,
+      };
```

Comment:
The current selection of the PropertiesWebSocketContainer cannot be null anymore so it does not have the same behavior when there is no selection